4-order-api-start/pkg/middleware/logs.go

```diff
@@ -0,0 +1,27 @@
+package middleware
+
+import (
+	log "github.com/sirupsen/logrus"
+	"net/http"
+	"time"
+)
+
+func Logging(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		start := time.Now()
+		wrapper := &WrapperWriter{
+			ResponseWriter: w,
+			StatusCode:     http.StatusOK,
+		}
+		next.ServeHTTP(wrapper, r)
+		log.WithFields(log.Fields{
+			"statusCode": wrapper.StatusCode,
+			"method":     r.Method,
+			"path":       r.URL.Path,
+			"query":      r.URL.RawQuery,
+			"ip":         r.RemoteAddr,
+			"user-agent": r.UserAgent(),
+			"elapsed":    time.Since(start),
```

Comment:
```suggestion
			"ip":         r.RemoteAddr,
			"user-agent": r.UserAgent(),
			"elapsed":    time.Since(start),
			"userAgent":     userAgent,
			"responseSize":  responseSize,
			"requestID":     r.Context().Value("requestID"),
```

Reply:
Давай те укажем чуть больше информации